### ISSUE TITLE
[OUDS] using font icon for prefix in Text Input doesn't works on built documentation

### ISSUE THREAD

**Issue by vprothais** [Orange-OpenSource/Orange-Boosted-Bootstrap#3204]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In the built and published documentation of Text Input the loading of the font-icon for the prefix icon is broken.
We though it should be fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/3195 but it is not.

### Reduced test cases

Check example in documentation : https://ouds-main--boosted.netlify.app/orange/docs/0.5/forms/text-input/#input-prefix-and-suffix

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

ouds main

**Comment by vprothais** [comment]
No icon for prefix in text input

**Comment by B3nz01d** [comment]
Issue closed during Sprint # 24
